velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -4530,6 +4530,12 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
   EXPECT_EQ(
       Timestamp(-66600, 0), dateParse("1969-12-31+11:00", "%Y-%m-%d+%H:%i"));
 
+  setQueryTimeZone("America/Los_Angeles");
+  // Tests if it uses weekdateformat if %v not present but %a is present.
+  EXPECT_EQ(
+      Timestamp(1730707200, 0),
+      dateParse("04-Nov-2024 (Mon)", "%d-%b-%Y (%a)"));
```

Comment:
does Joda throw is the day of the week doesn't match the rest of the date being parsed?

Reply:
No joda doesnt throw , the behavior depends on the specifier  ; Its why we need a fuzzer to ensure we are in sync and have behavior similar to that shown by Joda.